contracts/staking/contracts/test/TestDelegatorRewards.sol

```diff
@@ -40,7 +40,9 @@ contract TestDelegatorRewards is
     }
 
     constructor() public {
+        _addAuthorizedAddress(msg.sender);
```

Comment:
Especially after seeing this, preventing non-fresh initialization may be the move. This said, it would be one more thing to consider when re-deploying the staking logic contract, which may be something to weigh for this decision.